test/send.js

```diff
@@ -702,9 +702,11 @@ describe('send(file).pipe(res)', function () {
         .pipe(res)
       })
 
+      var filePath = path.join(fixtures, 'pets', 'index.html')
+
       request(app)
       .get('/pets/')
-      .expect(200, fs.readFileSync(path.join(fixtures, 'pets', 'index.html'), 'utf8'), done)
+      .expect(200, fs.readFileSync(filePath, 'utf8'), done)
```

Comment:
Revert the change if you would like this landed... Please don't include unrelated changes. If we wanted to enforce a max line length, we would do so, but we do not.